requirements.txt

```diff
@@ -5,6 +5,7 @@ idna==2.10
 isort==5.4.2
 lazy-object-proxy==1.4.3
 mccabe==0.6.1
+numpy==1.19.1
```

Comment:
Can you merge these two commits into a single commit?
Without evaluator.py, this change is meaningless, while we want all commits to be meaningful.
Don't commit a change until we really need it.

Reply:
In rebase motion, change _pick_ to _fixup_ can achieve this target.

Reply:
opps, I used '''squash'''.... Is that ok?

Reply:
Yes, but you must refine the commit message.
Do it by `git commit --amend`. The main subject of your commit should be about evaluator instead of numpy package.